kfdef/kfctl_grafana.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: kfdef.apps.kubeflow.org/v1
+kind: KfDef
+metadata:
+  name: grafana
+  namespace: opendatahub
+spec:
+  applications:
+    - name: odh-common
+      kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: odh-common
+    - name: prometheus-cluster
+      kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: prometheus/cluster
+    - name: prometheus-operator
+      kustomizeConfig:
+        overlays:
+          - kubeflow
+        repoRef:
+          name: manifests
+          path: prometheus/operator
+    - name: grafana-cluster
+      kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: grafana/cluster
+    - name: grafana-instance
+      kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: grafana/grafana
+  repos:
+    - name: manifests
+      uri: https://github.com/opendatahub-io/odh-manifests/tarball/master
+  version: master
```

Comment:
We should not have a kfdef just for installing grafana. We have a default kfdef that includes grafana and users can delete/add any app they need.

Reply:
Also uri should not point to master.